libs/pmp-web/repository/repository-settings/feature/src/lib/containers/settings/common-settings/common-settings.component.scss

```diff
@@ -0,0 +1,25 @@
+.half-width-container {
+  width: 50%;
+  margin: auto;
+}
+
+.settings-container {
+  @extend .half-width-container;
```

Comment:
In relation to my comment above, this would then be @include <mixin-name> instead of @extend <class-name>.